Flakiness when running tests against a local auth server

Seen this a few times in travis today:

```
> fxa-js-client@0.1.57 test-local /home/travis/build/mozilla/fxa-js-client
> intern-client config=tests/intern auth_server=LOCAL

Running against http://127.0.0.1:9000
group: main
group: account
FAIL: #destroy (84ms)
AssertionError: Account is gone
  at <tests/lib/account.js:52:22>
  at HandleCallback  <components/p/p.js:502:12>
  at Then  <components/p/p.js:491:4>
  at Immediate.flush [as _onImmediate]  <components/p/p.js:207:4>
  at processImmediate [as _immediateCallback]  <timers.js:396:17>
FAIL: #keys (45ms)
AssertionError: expected { error: 'Timeout error', errno: 999 } to be falsy
  at assert.notOk  <node_modules/intern-geezer/assert.js:493:10>
  at HandleCallback  <components/p/p.js:502:12>
  at Then  <components/p/p.js:491:4>
  at Immediate.flush [as _onImmediate]  <components/p/p.js:207:4>
  at processImmediate [as _immediateCallback]  <timers.js:396:17>
FAIL: #destroy with incorrect case (18ms)
AssertionError: expected 999 to equal 102
  at <tests/lib/account.js:106:20>
  at HandleCallback  <components/p/p.js:502:12>
  at Then  <components/p/p.js:491:4>
  at Immediate.flush [as _onImmediate]  <components/p/p.js:207:4>
  at processImmediate [as _immediateCallback]  <timers.js:396:17>
```

First test fails with `Account is gone`, subsequent tests fail with timeouts.

--- a/tests/ci/travis-auth-server-test.sh
+++ b/tests/ci/travis-auth-server-test.sh
@@ -5,7 +5,7 @@ git clone https://github.com/mozilla/fxa-auth-server.git
 cd fxa-auth-server && npm i
 SECONDARY_EMAIL_ENABLED=true SIGNIN_CONFIRMATION_ENABLED=true SIGNIN_CONFIRMATION_FORCE_EMAIL_REGEX="^confirm.*@restmail\\.net$"  SIGNIN_UNBLOCK_ALLOWED_EMAILS="^block.*@restmail\\.net$" SIGNIN_UNBLOCK_FORCED_EMAILS="^block.*@restmail\\.net$" npm start &
 cd ..
-sleep 5
+sleep 10
 
 # Run the tests against the local auth server
 npm run test-local

